src/components/Agreed/__tests__/Agreed.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+
+import Agreed from '../Agreed';
+
+describe('COMPONENT - Agreed', () => {
+  it('renders correctly', () => {
+    const component = create(<Agreed />);
+
+    expect(component.toJSON()).toMatchSnapshot();
```

Comment:
it actually has two states, one for media tablet another one for web - I would test css here as well.